src/pages/Results-Basic.tsx

```diff
@@ -77,6 +78,7 @@ export function ResultsBasic():JSX.Element {
                 <button onClick={() => navigate('/')}>Back to Home</button>
                 <div className='results-header'><h1 className='results-header-text'>Results</h1></div>
             </div>
+            {/* There was some use of chatGPT here although it was modified by hand to add in  */}
```

Comment:
Not sure what you mean by the end of this, but you need to state what specifically is used by chatGPT, like maybe the sendMessage button, because you need to follow it up with functionality